app/src/main/java/deakin/gopher/guardian/model/PatientStatus.java

```diff
@@ -0,0 +1,7 @@
+package deakin.gopher.guardian.model;
+
+public enum PatientStatus {
```

Comment:
Can this file be Kotlin instead? Should support enums the same

Reply:
Yup done.